WooCommerce/Classes/ViewModels/OrderDetailsViewModel/OrderPaymentDetailsViewModel.swift

```diff
@@ -82,12 +83,55 @@ final class OrderPaymentDetailsViewModel {
         return String.localizedStringWithFormat(template, styleDate, order.paymentMethodTitle)
     }
 
+    /// Refund Summary
+    /// - returns: A full sentence summary of the date the refund was created, which payment gateway it was refunded to, and a link to the detailed refund.
+    /// Example: Oct 28, 2019 via Credit Card (Stripe)
+    ///
+    var refundSummary: NSAttributedString? {
+        guard let refund = refund else {
+            return nil
+        }
+
+        // First, localize all the pieces of the sentence.
+        let dateCreated = DateFormatter.mediumLengthLocalizedDateFormatter.string(from: refund.dateCreated)
+
+        let hasRefundGateway = refund.isAutomated ?? false
+
+        // Yes, we're making the assumption that the payment method is the same as the refund method.
+        let refundType = hasRefundGateway ? order.paymentMethodTitle : NSLocalizedString(
+            "manual refund",
+            comment: "A manual refund is one where the store owner has given the purchaser alternative funds" +
+                " (cash, check, ACH) instead of using the payment gateway to create a refund " +
+                "(credit card or debit card was refunded)"
+        )
+
+        let viewDetailsText = NSLocalizedString("View details",
+                                                comment: "This text is linked so the user can view refund details.")
+
+        let template = NSLocalizedString("%@ via %@ – %@",
+                                         comment: "It reads: \"<date> via <refund method type> – View details\". The text `View details` is a link.")
+        let refundText = String.localizedStringWithFormat(template, dateCreated, refundType, viewDetailsText)
+
+        let refundAttrText = NSMutableAttributedString(string: refundText)
+        let location = refundAttrText.string.count - viewDetailsText.count
+        let viewDetailsRange = NSRange(location: location, length: viewDetailsText.count)
+
+         // Last, style the link text.
+        let linkAttributes: [NSAttributedString.Key: Any] = [
+            .foregroundColor: StyleManager.wooCommerceBrandColor
```

Comment:
let's use the new color in `UIColor+SemanticColors` from Cesar's Dark Mode color palette work! p99K0U-21F-p2

Reply:
thanks for the reminder +1